content/ja/docs/tasks/configmap-secret/managing-secret-using-kustomize.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Kustomizeを使用してSecretを管理する
+content_type: task
+weight: 30
+description: kustomization.yamlを使用してSecretを作成する
+---
+
+<!-- overview -->
+
+Kubernetes v1.14以降、`kubectl`は[Kustomizeを使ったオブジェクト管理](/docs/tasks/manage-kubernetes-objects/kustomization/)をサポートしています。
+KustomizeはSecretやConfigMapを作成するためのリソースジェネレーターを提供します。
+Kustomizeジェネレーターは、ディレクトリ内の`kustomization.yaml`ファイルで指定します。
+Secretを生成したら、`kubectl apply`でAPIサーバー上にSecretを作成します。
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}}
```

Comment:
This refers to `https://github.com/kubernetes/website/blob/master/content/ja/includes/task-tutorial-prereqs.md`.
There are some diffs between [`en`](https://github.com/kubernetes/website/blob/master/content/en/includes/task-tutorial-prereqs.md) and [`ja`](https://github.com/kubernetes/website/blob/master/content/ja/includes/task-tutorial-prereqs.md).

Current `ja` doc doesn't have a sentence of `It is recommended to run this tutorial on a cluster with at least two nodes......`

Should we update that file in this PR? :thinking:

Reply:
I'm ready to update the content, but should it be updated in v1.21 branch, which has not been created yet?

Reply:
I think it's OK to add all files for PR for `master` branch.
I refered [here for policy to localization](https://kubernetes.io/ja/docs/contribute/localization/#translate-new-page).